bokehjs/src/coffee/common/canvas.coffee

```diff
@@ -43,7 +67,7 @@ class CanvasView extends ContinuumView
                           @ctx.mozBackingStorePixelRatio ||
                           @ctx.msBackingStorePixelRatio ||
                           @ctx.oBackingStorePixelRatio ||
-                          @ctx.backingStorePixelRatio || 1
+                          @ctx.backingStorePixelRatio || 1  # ak: wtf?
```

Comment:
Or are you asking what the backing store pixel ratio is? In which case: http://www.html5rocks.com/en/tutorials/canvas/hidpi/